src/modules/dashboards/saga.ts

```diff
@@ -343,6 +349,69 @@ export function* persistDashboardsOrder({
   }
 }
 
+export function* cloneDashboard({
+  payload,
+}: ReturnType<typeof cloneDashboardAction>) {
+  const { dashboardId } = payload;
+
+  const blobApi = yield getContext(BLOB_API);
```

Comment:
As this is async operation that could fail - I believe it should also be in `try {} catch` block